bigbone/src/main/kotlin/social/bigbone/api/entity/data/Focus.kt

```diff
@@ -15,5 +15,9 @@ data class Focus(
     @SerialName("y")
     val y: Float
 ) {
+    init {
+        require(x in -1.0..1.0) { "x coordinate must be between -1.0 and +1.0 but was $x" }
+        require(y in -1.0..1.0) { "y coordinate must be between -1.0 and +1.0 but was $y" }
+    }
```

Comment:
Too bad that we don’t have the nice [`FloatRange` annotation the Android SDK offers](https://developer.android.com/reference/kotlin/androidx/annotation/FloatRange). 

Instead, I have added this check to the constructor so that library users cannot send out-of-bounds coordinates even if they didn’t read the kDoc note.